config/config.go

```diff
@@ -518,7 +518,9 @@ type CoProcessConfig struct {
 
 type CertificatesConfig struct {
 	API []string `json:"apis"`
-	// Specify upstream mutual TLS certificates at a global level in the following format: `{ "<host>": "<cert>" }``
+	// Specify upstream mutual TLS certificates at a global level as a map of domain -> certificate id or path
+	// In tyk.conf, this would be `"security": {"certificates": {"upstream": {"*": "/certs/default.pem", "example.com": "ab23ef123"}}},`; as an environment variable it would be
+	// `*:/certs/default.pem,example.com:ab23ef123`.
```

Comment:
```suggestion
	//  And if using environment variables you would set `TYK_GATEWAY_SECURITY_CERTIFICATES_UPSTREAM` to `*:/certs/default.pem,example.com:ab23ef123`.
```

Reply:
@andyo-tyk we are auto generating the env variable name here

Reply:
```suggestion
	// And if using environment variables you would set this to `*:/certs/default.pem,example.com:ab23ef123`.
```